### ISSUE TITLE
Dealing with deprecation notices when upgrading to Java 17

### ISSUE THREAD

**Issue by martinda** [cdancy/bitbucket-rest#375]
On the upgrade path to Java 17, there are some obstacles inside jclouds. Someone would like to do one more release of jclouds before jclouds is put in the attic, and it is being discussed [here](https://lists.apache.org/thread/z7lg1y0rjp2xlkxhkkg76190tx2lznjt).

I just wanted to capture that discussion here because it was not obvious to find and it might help with the deprecation notices that  are seen when compiling this library with `-Xlint:deprecation`.

**Comment by cdancy** [comment]
@martinda yeah I've been following along on the mailing list as well. Lets see what happens.

@martinda how are you are what's going in your world? Hopefully things are going well.
